Makefile

```diff
@@ -115,6 +111,8 @@ TAGS_EVIDENCE := $(MAKE_EVIDENCE_DIR)/tags
 
 TEST_TAGS ?= sqlite sqlite_unlock_notify
 
+TAR_EXCLUDES := .git node_modules $(FOMANTIC_WORK_DIR)/node_modules $(DIST) $(MAKE_EVIDENCE_DIR) $(AIR_TMP_DIR)
```

Comment:
```suggestion
TAR_EXCLUDES := .git data node_modules $(FOMANTIC_WORK_DIR)/node_modules $(DIST) $(MAKE_EVIDENCE_DIR) $(AIR_TMP_DIR)
```

Reply:
I guess we can put even more such folders that may be present in a development setup using the default paths, like `indexers`and `queues` directories. Thought it's really a non-issue because CI builds from a clean slate anyways.